pyproject.toml

```diff
@@ -38,7 +39,11 @@ multi_line_output = 3 # Vertical Hanging Indent
 src_paths = "tap_github"
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
+# Uncomment the pinned version in favor of the git URL once
+# https://github.com/python-poetry/poetry-core/pull/257 is merged
+# and a new poetry-core 1.0.x is released
+# requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core @ git+https://github.com/python-poetry/poetry-core.git@master"]
```

Comment:
This lets us set `pip_url: -e .` in `meltano.yml` for editable installation 🙂

Reply:
This scares me, as you're not really pinning your build system version at all here. Is it on purpose?

Reply:
@laurentS  Me too 😅. I don't think anyone uses this meltano file is used in production, though so I'd be ok with referencing a branch and watching closely for a pypi release of poetry-core with pep 660 support. wdyt?